test/unittests/api_test.cpp

```diff
@@ -427,11 +432,13 @@ TEST(api, find_exported_function)
         "0061736d010000000105016000017f021001087370656374657374036261720000040401700000050401010102"
         "0606017f0041000b07170403666f6f000001670300037461620100036d656d0200");
 
-    auto bar = [](Instance&, const Value*, int) { return Value{42}; };
+    constexpr auto bar = [](std::any&, Instance&, const Value*, int) -> ExecutionResult {
+        return Value{42};
+    };
     const auto bar_type = FuncType{{}, {ValType::i32}};
 
     auto instance_reexported_function =
-        instantiate(parse(wasm_reexported_function), {{bar, bar_type.inputs, bar_type.outputs}});
```

Comment:
The inputs/outputs seems to be like a stray change, but looking at other cases below we do seem to have a constructor for it.

Reply:
Yes, it's not required / can be split out.

Reply:
Fine with me, it is only this case so having a single PR for it would be overkill.